lib/factory_ex/application.ex

```diff
@@ -0,0 +1,35 @@
+defmodule FactoryEx.Application do
+  @moduledoc false
+
+  use Application
+
+  @impl true
+  def start(_, _) do
+    FactoryEx.SchemaCounter.start()
+
+    children = [
+      FactoryEx.FactoryStore
```

Comment:
We shouldn't have an application.ex, instead you start test helpers in the test_helper.exs file